include/fmt/format-inl.h

```diff
@@ -1162,25 +1162,10 @@ FMT_INLINE int remove_trailing_zeros(uint64_t& n) noexcept {
 
   // Is n is divisible by 10^8?
   if ((nm.high() & ((1ull << (90 - 64)) - 1)) == 0 && nm.low() < magic_number) {
-    // If yes, work with the quotient.
+    // If yes, work with the quotient...
     auto n32 = static_cast<uint32_t>(nm.high() >> (90 - 64));
-
-    const uint32_t mod_inv_5 = 0xcccccccd;
-    const uint32_t mod_inv_25 = mod_inv_5 * mod_inv_5;
-
-    int s = 8;
```

Comment:
Looks like 8 here gets lost.

Reply:
totally right, I changed the signature of `remove_trailing_zeros(uint32_t& n, int s = 0)` in order to allow passing the s